versions.props

```diff
@@ -46,11 +48,3 @@ javax.annotation:javax.annotation-api = 1.3.2
 javax.ws.rs:javax.ws.rs-api = 2.1.1
 javax.inject:javax.inject = 1
 
-# dependency-upgrader:OFF
-
-com.fasterxml.jackson.*:* = 2.13.1
-
-# Pin to same version as spark
-io.dropwizard.metrics:metrics-core = 3.2.6
```

Comment:
When a new dropwizard-metrics version is released, we don't want this repo to to cause everything else to resolve it upward -- that has historically caused issues for spark.

Reply:
But that already happens - we're currently using a 4.x version which is being bumped by transitive dependencies. So this constraint is just confusing.

Reply:
the dw-metrics dep should ideally be produced by tritium, however at the limit we'd end up splitting out that dependency into a separate excavator if we ran into that sort of api split again